arelle/resources/cache/http/www.xbrl.org/2013/inlineXBRL/xhtml-inlinexbrl-1_1.xsd

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
We should include both the 1.1 (http://www.xbrl.org/2013/inlineXBRL/xhtml-inlinexbrl-1_1.xsd) and 1.0 (http://www.xbrl.org/2008/inlineXBRL/xhtml-inlinexbrl-1_0.xsd) inline schemas.

Also, I'm inclined to think these should work for both http and https URLs.

Reply:
If enabled, the `httpRedirect` feature would apply to the built-in cache as well. But if you're suggesting that we should have http/https be essentially equivalent by default in the built-in cache, I can make it so.